### DISCUSSION TITLE
第2节课笔记(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#221] [笔记]
20班的宝子们，笔记在这个下面哦

**Comment by liu673** [comment]
第二节课笔记加作业：https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_2/lesson_2_note.md

**Comment by deyouzuishuai** [comment]
笔记：
https://zhuanlan.zhihu.com/p/677134650

**Comment by TtKkD** [comment]
http://t.csdnimg.cn/h5LRz

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135563168

**Comment by TLzh** [comment]
和作业同样的链接
https://blog.csdn.net/zhiho14/article/details/135541245

**Comment by Walshyu** [comment]
https://github.com/Liltomato1998/hello-world/blob/main/LLM/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by zhangyiyuan1112** [comment]
https://iqxyis6ody.feishu.cn/docx/S22bdAFPnoLoX7xp6dhcyWGfnYg?from=from_copylink

**Comment by qiaoxiaopang** [comment]
https://blog.csdn.net/weixin_74322438/article/details/135575057

**Comment by forestemperor** [comment]
https://github.com/InternLM/tutorial/discussions/221

**Comment by pooh0315** [comment]
https://blog.csdn.net/m0_74206611/article/details/135586108

**Comment by yalunren** [comment]
https://docs.qq.com/doc/DZHNnUXVtY25NdkFo?u=ccf8085364c54d099fd089ceaab4fa20

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679656393
